### PR TITLE
ENH: Actually Use Config Files

### PR DESCRIPTION
**Pull request by ZLLentz**
This required other big changes, like loading devices independently, which makes the GUI more robust. This also involved changes/additions to the config loader, and an equivalent sim config loader for when we open the gui in sim mode.

### CODE REVIEW

File: skywalker/gui.py [hunk 1]
```diff
@@ -17,15 +17,13 @@
                               QObject, QEvent)
 from pydm.PyQt.QtGui import QDoubleValidator, QDialog
 
-from pcdsdevices import sim
 from pcdsdevices.epics.attenuator import FeeAtt
```
Comment (teddyrendahl) [inline skywalker/gui.py:20]: Should we load this from configuration as well?

**Review by teddyrendahl — APPROVED** [review]

**Review by teddyrendahl — APPROVED** [review]
Fine with all this, as long as tests are fixed.

**Comment by teddyrendahl** [comment]
Afterwards we can https://github.com/slaclab/pswalker/pull/65!

File: skywalker/gui.py (hunk 1 above)
Comment (ZLLentz) [reply skywalker/gui.py:20]: Not sure, it can be a next step. Right now there's a hard-coded simulated config.

**Pull request merged**